src/js/edu-benefits/components/benefits-eligibility/BenefitsRelinquishmentFields.jsx

```diff
@@ -19,33 +59,16 @@ export default class BenefitsRelinquishmentFields extends React.Component {
               additionalClass="edu-benefits-active-group"
               open={showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)}>
             <div>
-              <ErrorableRadioButtons required={this.props.data.chapter33}
+              <ErrorableRadioButtons
+                  required={this.props.data.chapter33}
                   errorMessage={validateIfDirty(this.props.data.benefitsRelinquished, isNotBlank) ? '' : 'Please select a response'}
                   label="I choose to give up:"
                   name="benefitsRelinquished"
-                  options={relinquishableBenefits}
+                  options={options}
                   value={this.props.data.benefitsRelinquished}
-                  onValueChange={(update) => {this.props.onStateChange('benefitsRelinquished', update);}}/>
-            </div>
-            <div>
-              <DateInput required={showRelinquishedEffectiveDate(this.props.data.benefitsRelinquished.value)}
-                  allowFutureDates
-                  errorMessage="Date cannot be earlier than 2 years ago"
-                  validation={validateIfDirtyDateObj(this.props.data.benefitsRelinquishedDate, isValidRelinquishedDate)}
-                  label="Effective date"
-                  name="benefitsRelinquishedDate"
-                  day={this.props.data.benefitsRelinquishedDate.day}
-                  month={this.props.data.benefitsRelinquishedDate.month}
-                  year={this.props.data.benefitsRelinquishedDate.year}
-                  onValueChange={(update) => {this.props.onStateChange('benefitsRelinquishedDate', update);}}/>
-              <div>
-                <ul>
-                  <li>Use today’s date unless you aren’t going to use your Post 9/11 GI Bill benefits until later.</li>
-                  <li>If you pick a future date, you can’t get benefits until then.</li>
-                  <li>If your classes started less than 2 years ago, enter the date they began.</li>
-                </ul>
-              </div>
+                  onValueChange={(update) => { this.props.onStateChange('benefitsRelinquished', update); }}/>
             </div>
+            <div></div>
```

Comment:
This div is needed for ExpandingGroup to work properly.  I know, it's a little weird...